.github/workflows/build-publish-lib.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and Publish the Library
+on:
+  push:
+    tags:
+      - v*
```

Comment:
I suggested that you look at the geocoder example and probably you have the trigger from there. But actually this is not the best way of doing it because it can result in inconsistencies between the git version tag and the version in package.json. A better way is to use manual workflow trigger, take the version from package.json, and tag the git commit from inside the github workflow. Please have a look how we do it in MapLibre GL JS: https://github.com/maplibre/maplibre-gl-js/blob/main/.github/workflows/release.yml

Reply:
Makes sense. Will give it a look later.

Reply:
@wipfli updated the workflow accordingly. Tested on the fork, seems to work just fine up until the "Deploy" section (for obvious reasons).